lib/response_bank/middleware.rb

```diff
@@ -37,12 +37,21 @@ def call(env)
           body_compressed = nil
           if body_string && body_string != ""
             headers['Content-Encoding'] = content_encoding
-            body_compressed = ResponseBank.compress(body_string, content_encoding)
+            env["cacheable.compression_level"] ||= ResponseBank.compression_level_for(content_encoding)
+            time = ResponseBank.measure do
+              body_compressed = ResponseBank.compress(
+                body_string,
+                content_encoding,
+                compression_level: env["cacheable.compression_level"],
+              )
+            end
+            ResponseBank.log("Compression time: #{time}ms")
+            env["cacheable.compression_time"] = time
           end
 
           cached_headers = headers.slice(*CACHEABLE_HEADERS)
           # Store result
-          cache_data = [status, cached_headers, body_compressed, timestamp]
+          cache_data = [status, cached_headers, body_compressed, timestamp, env["cacheable.compression_level"]]
```

Comment:
Why do we change the format of the cache data? Is it backward compatible with the existing cache during deployment?